Classes/Controllers/PBGitHistoryController.m

```diff
@@ -401,6 +401,21 @@ - (IBAction)selectPrevious:(id)sender
 	[searchController selectPreviousResult];
 }
 
+- (IBAction) selectParentCommit:(id)sender
+{
+	NSArray *selectedObjects = commitController.selectedObjects;
+	if (selectedObjects.count != 1) return;
+
+	PBGitCommit *selectedCommit = selectedObjects[0];
+
+	NSArray <GTOID *> *parents = selectedCommit.parents;
+	/* TODO: This is a merge commit. It would be nice to choose the parent with
+	 * the most commits, but for now we will use whatever commit is our first parent.
+	 */
+
```

Comment:
The first parent is [the branch being merged into, for normal merges][1].

[1]: https://stackoverflow.com/questions/18301284/git-commit-parents-order/18301726#18301726